plugins/nodes/src/js/columns/NodesTableIpColumn.tsx

```diff
@@ -7,10 +7,11 @@ import { Link } from "react-router";
 import { Tooltip } from "reactjs-components";
 import Icon from "#SRC/js/components/Icon";
 import { TextCell } from "@dcos/ui-kit";
+import ipToInt from "ip-to-int";
```

Comment:
It doesn't support ipv6 and there's no related tradeoff. Please verify that ipv6 is not a requirement.

Reply:
Would you rather drop the improvement? I checked with customer support, there are customers on ipv6, so they would fall back to the string sorting.